lib/improver/nbhood/nbhood.py

```diff
@@ -808,7 +167,8 @@ class NeighbourhoodProcessing(object):
     """
 
     def __init__(self, neighbourhood_method, radii, lead_times=None,
-                 unweighted_mode=False, ens_factor=1.0):
+                 weighted_mode=True, ens_factor=1.0,
+                 percentiles=PercentileConverter.DEFAULT_PERCENTILES):
```

Comment:
Caroline was going to move this outside the class, but still in that module. She reverted it to avoid creating a conflict with this PR.

Should we use a new PR to make this change?